index.html

```diff
@@ -43,35 +43,36 @@
 
       <div class="row justify-content-center">
         <div class="col-xl-9 d-flex align-items-center justify-content-between">
-          <h1 class="logo"><a href="index.html">KnightOne</a></h1>
+          <h1 class="logo"><a href="index.html">ProffNITT</a></h1>
           <!-- Uncomment below if you prefer to use an image logo -->
           <!-- <a href="index.html" class="logo"><img src="assets/img/logo.png" alt="" class="img-fluid"></a>-->
 
           <nav class="nav-menu d-none d-lg-block">
             <ul>
               <li class="active"><a href="index.html">Home</a></li>
               <li><a href="#about">About</a></li>
-              <li><a href="#services">Services</a></li>
-              <li><a href="#portfolio">Portfolio</a></li>
-              <li><a href="#pricing">Pricing</a></li>
-              <li class="drop-down"><a href="">Drop Down</a>
+              <li><a href="#services">Members</a></li>
+              <li><a href="#portfolio">BharatX</a></li>
+              <!-- <li><a href="#pricing">Pricing</a></li> -->
+              <li class="drop-down"><a href="">Events</a>
                 <ul>
-                  <li><a href="#">Drop Down 1</a></li>
-                  <li class="drop-down"><a href="#">Deep Drop Down</a>
+                  <li><a href="#">INVESTAS</a></li>
+                  <!-- <li class="drop-down"><a href="#">Deep Drop Down</a>
                     <ul>
                       <li><a href="#">Deep Drop Down 1</a></li>
                       <li><a href="#">Deep Drop Down 2</a></li>
                       <li><a href="#">Deep Drop Down 3</a></li>
                       <li><a href="#">Deep Drop Down 4</a></li>
                       <li><a href="#">Deep Drop Down 5</a></li>
                     </ul>
-                  </li>
-                  <li><a href="#">Drop Down 2</a></li>
-                  <li><a href="#">Drop Down 3</a></li>
-                  <li><a href="#">Drop Down 4</a></li>
+                  </li> -->
+                  <li><a href="#">INSIGHT</a></li>
+                  <li><a href="#">THE TALK</a></li>
+                  <li><a href="#">GUEST LECTURES AND WEBINARS</a></li>
                 </ul>

```

Comment:
The `href`s feel incorrect. Change it according to the content. Later when others work on the sections, they can refer to the navbar and decide the section ID.